docs/modelserving/v1beta1/lightgbm/README.md

```diff
@@ -263,13 +263,35 @@ curl -v \
 ### Create the InferenceService with gRPC endpoint
 Create the inference service yaml and expose the gRPC port, currently only one port is allowed to expose either HTTP or gRPC port and by default HTTP port is exposed.
 
-=== "Yaml"
+!!! Note
+    Currently, KServe only supports exposing either HTTP or gRPC port. By default, HTTP port is exposed.
 
+=== "Serverless"
     ```yaml
     apiVersion: "serving.kserve.io/v1beta1"
     kind: "InferenceService"
     metadata:
-      name: "lightgbm-v2-iris"
+      name: "lightgbm-v2-iris-grpc"
+    spec:
+      predictor:
+        model:
+          modelFormat:
+            name: lightgbm
+          protocolVersion: v2
+          runtime: kserve-lgbserver
+          storageUri: "gs://kfserving-examples/models/lightgbm/v2/iris"
+          ports:
+            - name: h2c          # knative expects grpc port name to be 'h2c'
```

Comment:
Is this something new? Knative requires the port name to be h2c

Reply:
Not at all. Here is the release note from knative. I’m happy to say that with the v0.4 release Knative now also [supports](https://www.knative.dev/docs/serving/samples/grpc-ping-go) both HTTP2 and gRPC for ports named h2c. https://knative.dev/blog/releases/announcing-knative-v0-4-release/. https://github.com/knative/specs/blob/main/specs/serving/runtime-contract.md#network-environment

Reply:
you changed the port name from h2c to gRPC-port in the examples

Reply:
I think you are referring to the raw deployment yaml. For raw deployment i am using 'grpc-port' as the port name. In Istio we need to specify the protocol name for grpc. For severless deployment i am using 'h2c' as the port name which is required by knative.